modules/hibernate-core/src/main/java/org/apache/ignite/cache/hibernate/HibernateAccessStrategyFactory.java

```diff
@@ -92,8 +95,23 @@
      * @param eConverter Exception converter.
      */
     HibernateAccessStrategyFactory(HibernateKeyTransformer keyTransformer, HibernateExceptionConverter eConverter) {
+        this(keyTransformer, eConverter, null);
+    }
+
+    /**
+     * @param keyTransformer Key transformer.
+     * @param eConverter Exception converter.
+     * @param ignite Ignite instance. Can be {@code null}, but in this case {@link #IGNITE_INSTANCE_NAME_PROPERTY}
+     *               or {@link #GRID_CONFIG_PROPERTY} should be provided via hibernate properties.
+     */
+    public HibernateAccessStrategyFactory(
```

Comment:
Why is this change needed? I can't find any usages of this constructor....

Reply:
I am going to use it in the JHipster integration